assets/sass/imports/_presentations.scss

```diff
@@ -0,0 +1,55 @@
+.presentation {
+    position: relative; 
+    display: flex; 
+    flex-direction: column;
+    justify-content: start;  
+    margin-left:-40%;
+    margin-right:-40%;
+    padding: 3%;
```

Comment:
You should always use absolute units (px) for padding and margins unless relative is necesssary. On 1st look, it seems percentages will make your layout responsive but it creates problems because the ratios are not constant w.r.t. to devic for e.g. on desktop, 20px padding looks good but on mobile, 8px looks good. Moreover unintended behaviour may occur due to [this](https://stackoverflow.com/questions/11003911/why-are-margin-padding-percentages-in-css-always-calculated-against-width)